### PR TITLE
Elections 2020 - 2021

### PR DESCRIPTION
**Pull request by iDuckDark**
Please add security @silverAndroid

### CODE REVIEW

File: backend/routes/vote.js [hunk 1]
```diff
@@ -14,6 +14,18 @@ router.get('/', (req, res, next) => {
     return handleSuccess(res, `Success`, { result: results });
 });
 
+router.get('/results', (req, res, next) => {
```
Comment (silverAndroid) [inline backend/routes/vote.js]: Can you remove this route

File: frontend/src/actions/allActions.jsx [hunk 2]
```diff
@@ -19,6 +19,23 @@ export function getUsers() {
     };
 }
 
+export function getElectionResults() {
```
Comment (silverAndroid) [inline frontend/src/actions/allActions.jsx]: Same with this action

File: frontend/src/data/execs-2020-2021.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    },
+    {
+        "name": "VACANT",
+        "position": "McNaughton Centre Director",
+        "image": "../../static/images/execs-2020-2021/mcnaughton-director.jpg",
+        "email": "mcnaughton@ieeeuottawa.ca"
+    },
+    {
+        "name": "VACANT",
+        "position": "Photonics Chair ",
```
Comment (silverAndroid) [inline frontend/src/data/execs-2020-2021.json]: extra space

File: frontend/src/data/execs-2020-2021.json [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        "image": "../../static/images/execs-2020-2021/mcnaughton-director.jpg",
+        "email": "(Grad Students Only)"
+    },
+    {
+        "name": "VACANT",
+        "position": "Photonics Vice-Chair ",
```
Comment (silverAndroid) [inline frontend/src/data/execs-2020-2021.json]: extra space

File: frontend/src/pages/Elections/ElectionResults.jsx [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+import { Typography, Card, CardContent } from '../../helpers/material-ui';
+import { electionResults } from '../../helpers/elections';
+
+class ElectionResults extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { electionResults };
+    }
+
+    // componentDidMount() {
```
Comment (silverAndroid) [inline frontend/src/pages/Elections/ElectionResults.jsx]: Probably should remove this commented code

File: frontend/src/utils/routes.js [hunk 6]
```diff
@@ -24,15 +32,25 @@ const routes = [
...
-    {
-        title: 'Election Platforms',
-        link: '/Elections/ElectionPlatforms'
+        items: [
+            {
+                title: 'Election Info',
+                link: '/Elections/Elections',
+                exact: '/elections'
+            },
+            // {
```
Comment (silverAndroid) [inline frontend/src/utils/routes.js]: Is this route still needed?

Comment (iDuckDark) [reply frontend/src/utils/routes.js]: nope will remove

**Pull request merged**